docs/docs/api/03-Filters/customelement.md

```diff
@@ -3,9 +3,38 @@ displayed_sidebar: apiSidebar
 ---
 # customElement
```

Comment:
I know this is tempting to do right here. But you have to change the API Docs in the askui-inference repository :/